tree/tree/inc/TLeaf.h

```diff
@@ -72,8 +72,8 @@ class TLeaf : public TNamed {
    Int_t            fLen;             ///<  Number of fixed length elements in the leaf's data.
    Int_t            fLenType;         ///<  Number of bytes for this data type
    Int_t            fOffset;          ///<  Offset in ClonesArray object (if one)
-   Bool_t           fIsRange;         ///<  (=kTRUE if leaf has a range, kFALSE otherwise).  This is equivalent to being a 'leafcount'.  For a TLeafElement the range information is actually store in the TBranchElement.
-   Bool_t           fIsUnsigned;      ///<  (=kTRUE if unsigned, kFALSE otherwise)
+   bool             fIsRange;         ///<  (=true if leaf has a range, false otherwise).  This is equivalent to being a 'leafcount'.  For a TLeafElement the range information is actually store in the TBranchElement.
+   bool             fIsUnsigned;      ///<  (=true if unsigned, false otherwise)
```

Comment:
FYI @pcanal, this change of using `true/false` instead of `kTRUE/kFALSE` was causing the test to fail. Because the doc strings were written into the file, and since they are different the file size changed. I will therefore not change this in this PR.